pkg/channel/consolidated/status/status.go

```diff
@@ -361,15 +361,13 @@ func (m *Prober) RefreshPodProbing(ctx context.Context) {
 					zap.Any("subscription", sub.UID))
 				return
 			}
-			if !state.probedPods.Equal(target.PodIPs) {
```

Comment:
just so I understand, why was this condition no longer needed?

Reply:
To force refreshing the Pods regardless of whether the IPs changed or not. We can't for sure know if pods with the same IPs are the same ones or not.